src/frontend/src/pages/GanttPage/GanttPageWrapper.tsx

```diff
@@ -9,25 +9,70 @@ import ErrorPage from '../ErrorPage';
 import { Task } from './GanttPackage/types/public-types';
 import PageTitle from '../../layouts/PageTitle/PageTitle';
 import { Project, WbsElementStatus, WorkPackage } from 'shared';
-import GanttPage from './GanttPage';
+import GanttChart from './GanttChart';
 import { projectWbsPipe, wbsPipe } from '../../utils/pipes';
 import GanttPageFilter from './GanttPageFilter';
 import { ChangeEvent, FC, useEffect, useMemo, useState } from 'react';
 import { SelectChangeEvent } from '@mui/material/Select';
 import { useQuery } from '../../hooks/utils.hooks';
 import { useHistory } from 'react-router-dom';
-import { filterGanttProjects, buildGanttSearchParams, GanttFilters, sortWbs } from '../../utils/gantt.utils';
+import { filterGanttProjects, buildGanttSearchParams, GanttFilters } from '../../utils/gantt.utils';
 import { routes } from '../../utils/routes';
 import { useToast } from '../../hooks/toasts.hooks';
+import { Box, createTheme, Paper, ThemeProvider, Typography } from '@mui/material';
+import { nerThemeOptions, lightThemeOptions } from '../../utils/themes';
 
-interface GanttDisplayProject extends Project {
-  displayOrder: number;
-}
+const NO_TEAM = 'No Team';
 
-interface GanttDisplayWorkPackage extends WorkPackage {
-  displayOrder: number;
+interface GanttTask extends Task {
+  teamName: string;
 }
 
+const transformWorkPackageToGanttTask = (workPackage: WorkPackage, teamName: string): GanttTask => {
```

Comment:
maybe put these two transformation methods in a utils file, this file is getting pretty long